src/livePreview/live-preview.ts

```diff
@@ -100,7 +100,7 @@ export default class LivePreview {
 
         //! TODO: we replaced the handleOnChange() with this.
         //! I don't think we need this. Confirm and remove it.
-        config.onChange();
+        config.onChange!();
```

Comment:
When we will fix the types, let's ensure we remove this change as well. It may introduce bugs in the future.